contributions/catalog/controllers/contribute.py

```diff
@@ -344,6 +470,48 @@ def create():
     return render_template('contribute/contribute.html', required_capabilities=required_capability_list,
                            post=json_contribute, user=session["name"],  token=session['oauth_token']['access_token'])
 
+# reviewers page
+@bp.route('/contributions/reviwers', methods=['GET'])
+@login_required
+def reviewers_main():
```

Comment:
```suggestion
def reviews_main():
```
Similarly, please rename other code references to the previous `reviewers`  page to the `reviews` page. Thanks.

Reply:
changed

Reply:
Thanks.